src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -1403,6 +1403,10 @@ private BoundExpression GetDefaultParameterValue(SyntaxNode syntax, ParameterSym
                 // so replace the argument with default(int?).
                 defaultValue = new BoundDefaultExpression(syntax, parameterType) { WasCompilerGenerated = true };
             }
+            else if (defaultConstantValue.IsNull || defaultConstantValue.IsBad)
+            {
+                defaultValue = MakeLiteral(syntax, defaultConstantValue, parameterType, localRewriter);
```

Comment:
>defaultValue = MakeLiteral(syntax, defaultConstantValue, parameterType, localRewriter); [](start = 16, length = 87)

I think we would want to return BadExpression instead of null literal for ```defaultConstantValue.IsBad```. We wouldn't want to report any nullable warnings for that value, for example.